addons/web_editor/static/src/js/editor/odoo-editor/src/base_style.scss

```diff
@@ -8,24 +8,10 @@ li.oe-nested {
         padding: 0.5rem;
     }
 }
-$sizes: 'xs', 'sm', 'md', 'lg', 'xl', 'xxl';
 .o_text_columns {
     max-width: 100% !important;
     padding: 0 !important;
 }
-.o_text_columns > .row {
-    margin: 0 !important;
-    @each $size in $sizes {
-        @for $i from 1 through 12 {
-            & > .col-#{$size}-#{$i}:first-of-type {
-                padding-left: 0;
-            }
-            & > .col-#{$size}-#{$i}:last-of-type {
-                padding-right: 0;
-            }
-        }
-    }
-}
```

Comment:
Hello, did you make sure that removing this code did not introduce a regression towards the original issue that it was intended to fix? That is https://www.odoo.com/odoo/1695/tasks/3001988.

Reply:
@sobo-odoo Thank you! 👍